frontend/src/metabase/home/components/EmbedHomepage/FeedbackModal.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from "react";
+import { t } from "ttag";
+
+import {
+  Button,
+  Group,
+  Modal,
+  rem,
+  Stack,
+  Text,
+  Textarea,
+  TextInput,
+  Title,
+} from "metabase/ui";
+
+type FeedbackModalProps = {
+  opened: boolean;
+  onClose: () => void;
+  onSubmit: (feedback: { feedback: string; email: string }) => void;
+};
+
+export const FeedbackModal = ({
+  opened,
+  onClose,
+  onSubmit,
+}: FeedbackModalProps) => {
+  const [feedback, setFeedback] = useState("");
+  const [email, setEmail] = useState("");
+  const handleSubmit = () => onSubmit({ feedback, email });
+
+  return (
+    <Modal
+      size={rem(530)}
+      padding="xl"
+      opened={opened}
+      withCloseButton={false}
+      onClose={onClose}
+    >
+      <Title pb="sm">{t`How can we improve embedding?`}</Title>
+      <Stack spacing="lg">
+        {/* eslint-disable-next-line no-literal-metabase-strings -- only admins can see this component */}
+        <Text>{t`Please let us know what happened. We’re always looking for ways to improve Metabase.`}</Text>
+
+        <Textarea
+          label={t`Feedback`}
+          name="feedback"
+          placeholder={t`Tell us what happened`}
+          onChange={e => setFeedback(e.currentTarget.value)}
+          minRows={3}
+        />
+
+        <TextInput
```

Comment:
could we add `type="email"` to make sure we don't get weird emails?
![image](https://github.com/metabase/metabase/assets/1937582/84dc6e03-6cfd-4ed2-a5b5-e97eade01df8)

You would probably need to wrap all input in a form and submit via form, so we get the HTML form validation, but I think it's worth it.

Reply:
We ended up choosing to not validate anything in order to not add any friction, as this is already a point where a user could be frustrated and we're asking for precious feedback: https://metaboat.slack.com/archives/C063Q3F1HPF/p1713204269888089?thread_ts=1712319863.930869&cid=C063Q3F1HPF